docs/tulu1_tulu2.md

```diff
@@ -0,0 +1,105 @@
+## Released Checkpoints
```

Comment:
Could we add a little header, saying this page is the old tulu 1/2 documentation?

Reply:
could add the commit message here too to make it clear.